File: app/src/main/java/com/antonio/samir/meteoritelandingsspots/ui/meteoriteRecyclerView/selector/MeteoriteSelectorPortrait.java
package com.antonio.samir.meteoritelandingsspots.ui.meteoriteRecyclerView.selector;


import com.antonio.samir.meteoritelandingsspots.model.Meteorite;

public class MeteoriteSelectorPortrait implements MeteoriteSelector {

    private final MeteoriteSelectorView view;

    public MeteoriteSelectorPortrait(MeteoriteSelectorView view) {
        this.view = view;
    }

    @Override
    public void selectItemId(final Meteorite meteorite) {
        view.selectPortrait(meteorite);
    }
}

File: app/src/main/java/com/antonio/samir/meteoritelandingsspots/features/list/ui/recyclerView/selector/MeteoriteSelectorPortrait.kt
package com.antonio.samir.meteoritelandingsspots.features.list.ui.recyclerView.selector

import com.antonio.samir.meteoritelandingsspots.service.business.model.Meteorite


class MeteoriteSelectorPortrait(private val view: MeteoriteSelectorView) : MeteoriteSelector {

    override fun selectItem(meteorite: Meteorite) {
        view.selectPortrait(meteorite)
    }
}
